db/re/item_db.conf

```diff
@@ -61939,6 +61940,11 @@ item_db: (
 	Slots: 1
 	Loc: 256
 	View: 525
+	Script: <"
+		bonus bDex, 2;
+		skill TF_DOUBLE, 5;
```

Comment:
Well, let's check description again. This is kRO description (in korean) :

> 머리 위에서 잠이 든 귀여운 아기 용.
> 이대로 무얼하든 깨어나지 않을 것 같다.
> DEX+2
> 5레벨 더블어택 사용 가능.
> 더 높은 레벨을 배우고 있을 시, 해당 레벨로 발동.
> 계열 : 투구 방어 : 1
> 위치 : 상단 무게 : 10
> 요구 레벨 : 1
> 장착 : 전 직업

This is translation according to Google:

> For a cute baby sleeping on the head.
> No matter what I do, I will not wake up.
> DEX + 2
> Level 5 Double Attack available.
> If you are learning a higher level, activate it at that level.
> Category: Helm Defense: 1
> Location: Top Weight: 10
> Requires Level: 1
> Mounting: former occupation

And this is translation to propose to english, according to *athena format:

> A baby dragon who has been trained to help its master.
> DEX + 2
> Enables Level 5 Double Attack (When you learn a higher level, activate the corresponding level).
> Class: Headgear
> Defense: 1
> Location: Upper
> Weight: 10
> Level Requirement: 1
> Jobs: All classes

This headgear enables usage of Level 5 Double Attack with equipped weapon, but, if player has learned double attack at higher level, the chance is influenced by the level double attack is learned.

What you both propose is to enable Double Attack at level 5 or enable it at a higher level if player has already learned it at higher level. That is what that proposal script does and that doesn't have any sense, why would enable usage of a skill already learned?

There are several items who already brings Double Attack usage on actual database and are these:

```
1130	Nagan		skill TF_DOUBLE,5;	bonus bDoubleRate,25;
2820	Thief Manual	skill TF_DOUBLE,3;	bonus bDoubleRate,15;
4117	Sidewinder Card	skill TF_DOUBLE,1;	bonus bDoubleRate,5;
5283	Chick Hat	skill TF_DOUBLE,2;	bonus bDoubleRate,10;
5388	Snake Head	skill TF_DOUBLE,5;	bonus bDoubleRate,25;
13407	Refined Nagan	skill TF_DOUBLE,5;	bonus bDoubleRate,25;
```
When I say it's implicit on script I mean this `bonus bDoubleRate` activates it at learned level because effect gets influenced by the level of Double Attack learned, coming from the enabled skill by the gear or coming from naturally learned this skill.

I can't agree with you :( This has been the behaviour that this kind of gears have had since ever and I cannot change it.